### PR TITLE
Replacing individual tags with tag manager

### PR DESCRIPTION
**Pull request by manolingam**
Seems like a GA4 tag is also necessary and so I have added the config for it. Also, removed the trailing commas that was pushed previously due to prettier. Require your review @hammadj

### CODE REVIEW

**Comment by manolingam** [comment]
Instead of using multiple tags, I have added in tag manager for ease of use.

File: static/admin/index.html [hunk 1]
```diff
@@ -1,6 +1,26 @@
 <!DOCTYPE html>
 <html>
   <head>
+    <!-- Google Tag Manager -->
```
Comment (hammadj) [inline static/admin/index.html:4]: This page is only for the CMS, we dont need google analytics here, it wont track anything on the main wiki site.

We should be using the official GA plugin for Docusaurus instead: https://v2.docusaurus.io/docs/using-plugins/#docusaurusplugin-google-analytics

**Review by hammadj — CHANGES_REQUESTED** [review]
Should just use the GA plugin: https://v2.docusaurus.io/docs/using-plugins/#docusaurusplugin-google-analytics

File: static/admin/index.html (hunk 1 above)
Comment (manolingam) [reply static/admin/index.html:4]: Can that plugin be used to add google tag manager ID instead of analytics ID?

Comment (hammadj) [reply static/admin/index.html:4]: Yes there's one for tags as well: https://v2.docusaurus.io/docs/using-plugins/#docusaurusplugin-google-gtag

**Comment by luxumbra** [comment]
@manolingam do you have any objections to one of the newer players picking this up if you're busy raiding :crossed_swords: ?

**Comment by manolingam** [comment]
@luxumbra Sorry I missed to update. We have the analytics done and this was experimental to add a tag manager around it for GA4. But I think this is not necessary now. This one can be closed.

**Pull request closed without merging**